src/Mongo/test/Eventuous.Tests.Projections.MongoDB/ProjectWithBuilder.cs

```diff
@@ -58,18 +58,18 @@ public SutProjection(IMongoDatabase database)
             : base(database) {
             On<BookingImported>(
                 b => b
-                    .InsertOne
-                    .Document(
-                        (stream, e) => new BookingDocument(stream.GetId()) {
-                            RoomId       = e.RoomId,
-                            CheckInDate  = e.CheckIn,
-                            CheckOutDate = e.CheckOut,
-                            BookingPrice = e.Price,
-                            Outstanding  = e.Price
-                        }
-                    )
-            );
-
+                    .Bulk
```

Comment:
Shall we not have a completely new SUT for bulks?

Reply:
You are right.
Ideally, all the other operations (delete, deletemany, insertmany, updatemany) should be covered by tests.

Reply:
True. I now added Testcontainers to all the test projects, and fixed all the singleton fixtures. It should be easier to add tests, and I plan to cover more ops with tests, although it requires a bit of time. Tests are now running in PR jobs, so it will be safer now.

Reply:
Great addition!
But pls try not update those testcontainer image versions frequently. People with slow internet (me) will appreciate :)
It'll take some time for me to write those tests, but I'm on it.

Reply:
Judging default tags in the Testcontainers libs, I don't think they are obsessed with updates :) I only manually set the tag for ESDB container as I have to choose the alpha ARM build when working on my machine.